File: server/Models/Favorite.cs
namespace all_spice.Models;
public class Favorite
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string CreatorId { get; set; }
    public int RecipeId { get; set; }
    public string AccountId { get; set; }
    public Profile Creator { get; set; }
}

File: client/src/models/Favorite.js
export class Favorite {
    constructor(data) {
        this.id = data.id || data._id
        this.createdAt = new Date(data.createdAt)
        this.updatedAt = new Date(data.updatedAt)
        this.creatorId = data.creatorId
        this.creator = data.creator
        this.recipeId = data.recipeId
        this.accountId = data.accountId
        this.favoriteId = data.favoriteId
    }
}